custom/icds_reports/ucr/data_sources/child_delivery_forms.json

```diff
@@ -756,6 +756,15 @@
       }
     },
     "engine_id": "icds-ucr",
+    "sql_settings": {
+      "partition_config": [
+        {
+          "column": "timeend",
```

Comment:
Sorry if I wasn't clear in the email I sent out. Not all of these UCRs need to be partitioned and not all of them will have the same column name. For example, this UCR has a column_id `submitted_on` that is the equivalent of `form.meta.timeEnd`.

Also I don't think that we need to partition all of these. We only need to partition "large" data sources. What constitutes a "large" data source isn't exactly defined, but generally data sources with lots of cases ( like tasks, persons, child_health, ccs_record) and forms that are submitted very often (> 1 per day per AWC, so maybe data sources with more than 50M rows?)

I've been using a combination [Finding the total size of your biggest tables
](https://wiki.postgresql.org/wiki/Disk_Usage#Finding_the_total_size_of_your_biggest_tables) and the preview data source on ICDS to see which ones need it the most

Reply:
Yeah I didn't notice this requirement. Do you have this list somewhere? Or do you want me to find the sizes of all ICDS data sources?

Reply:
person_cases_v2
usage_forms
household_cases
gm_forms